src/olympia/scanners/templates/admin/scanners/scannerresult/formatted_matched_rules_with_files.html

```diff
@@ -12,8 +12,8 @@
     <td><a href="{% url 'admin:scanners_scannerrule_change' rule.pk %}">{{ rule.name }}</a></td>
     <td>
       {% for file in rule.files %}
-        {% if addon_id %}
-          <a href="{% url 'files.list' addon_id 'file' file %}">{{ file }}</a>
+        {% if file_id %}
+          <a href="{{ external_site_url }}{% url 'files.list' file_id 'file' file %}">{{ file }}</a>
```

Comment:
We've got `|absolutify` filter for this... though I'm not sure how we can use it here, since I assume this is a django template and not a jina one ?

Reply:
I saw the `absolutify` filter but I could not make it work :/

Reply:
Yeah let's not worry about it for now.